### DISCUSSION TITLE
[Q] Hide rows/columns which doesn't have data while writing to excel file?

### DISCUSSION THREAD

**Discussion by nitin-kapoor** [exceljs/exceljs#2450] [Q&A]
## 💬 Questions and Help Needed !!
Hide all rows/column which doesn't have data when writing data to excel file?
Any help will really be appreciated.

**How to hide all the rows and column which doesn't have data as highlighted in below Screenshot1**

![image](https://user-images.githubusercontent.com/34282001/82464058-3f601c00-9adb-11ea-8f17-485c8391944e.png)

**Want to achieve like Screenshot2**

![image](https://user-images.githubusercontent.com/34282001/82464156-5d2d8100-9adb-11ea-85ea-cfb0cc175080.png)

**Comment by nitin-kapoor** [comment]
Hi Alanscut,

What is the solution for this?
is this library provide this functionality? If "YES" then please help me with this.

**Comment by nitin-kapoor** [comment]
> ## 💬 Questions and Help Needed !!
> Hide all rows/column which doesn't have data when writing data to excel file?
> Any help will really be appreciated.
> 
> **How to hide all the rows and column which doesn't have data as highlighted in below Screenshot1**
> 
> ![image](https://user-images.githubusercontent.com/34282001/82464058-3f601c00-9adb-11ea-8f17-485c8391944e.png)
> 
> **Want to achieve like Screenshot2**
> 
> ![image](https://user-images.githubusercontent.com/34282001/82464156-5d2d8100-9adb-11ea-85ea-cfb0cc175080.png)

Any update on this?? Please help!!

**Comment by agupta93** [comment]
@Alanscut I would like to work on the above, if anybody is not already working on the same

**Comment by maxdeepfield** [comment]
any solution to this?
